google-cloud-bigquery/acceptance/bigquery/table_reference_test.rb

```diff
@@ -117,6 +118,30 @@
     partitioned_table.reload!
     partitioned_table.table_id.must_equal partitioned_table_id
     partitioned_table.time_partitioning_type.must_equal "DAY"
+    partitioned_table.time_partitioning_fied.must_be_nil
+    partitioned_table.time_partitioning_expiration.must_equal 1
+  end
+
+  it "gets and sets time partitioning by field" do
+    partitioned_table = dataset.table partitioned_table_id
+    if partitioned_table.nil?
+      partitioned_table = dataset.create_table partitioned_table_id do |updater|
+        updater.time_partitioning_type = "DAY"
+        updater.time_partitioning_field = "dob"
+        updater.time_partitioning_expiration = seven_days
+        updater.schema do |schema|
+          schema.timestamp "dob",   description: "dob description",   mode: :required
+        end
+      end
+    end
+
+    partitioned_table = dataset.table partitioned_table_id, skip_lookup: true
+    partitioned_table.time_partitioning_expiration = 1
+
+    partitioned_table.reload!
+    partitioned_table.table_id.must_equal partitioned_table_id
+    partitioned_table.time_partitioning_type.must_equal "DAY"
+    partitioned_table.time_partitioning_fied.must_equal "dob"
```

Comment:
NoMethodError: undefined method `time_partitioning_fied' for #<Google::Cloud::Bigquery::Table:0x00007fdcbb5b4e98>
Did you mean?  time_partitioning_field